pallets/pallet-bonded-coins/src/traits.rs

```diff
@@ -27,3 +56,20 @@ pub trait ResetTeam<AccountId>: Inspect<AccountId> {
 		freezer: AccountId,
 	) -> DispatchResult;
 }
+
+impl<T> ResetTeam<AccountIdOf<T>> for AssetsPallet<T>
+where
+	T: AssetConfig,
+	<<T as frame_system::Config>::Lookup as StaticLookup>::Source: From<AccountIdOf<T>>,
+{
+	fn reset_team(
+		id: Self::AssetId,
+		owner: AccountIdOf<T>,
+		admin: AccountIdOf<T>,
+		issuer: AccountIdOf<T>,
+		freezer: AccountIdOf<T>,
+	) -> DispatchResult {
+		let origin = RawOrigin::Signed(owner);
+		AssetsPallet::<T>::set_team(origin.into(), id.into(), issuer.into(), admin.into(), freezer.into())
```

Comment:
That's not quite how the `reset_team` trait works on the assets pallet versions that have it implemented - `owner` actually sets the new owner. I think you should either ignore the owner parameter alltogether, or call `transfer_ownership` in case the `owner` is different from the actual owner. In any case, you need to look up the owner similarly to how you look up the freezer in the `freeze` trait.

Reply:
[7300dd5](https://github.com/KILTprotocol/kilt-node/pull/780/commits/7300dd568781bd22468f65256a3d4c4fea813c64)